database/src/main/java/database/service/SqlDatabaseService.java

```diff
@@ -204,6 +256,36 @@ public boolean removeRecordFromTable(Class<? extends BaseEntity> entityClass, In
         }
     }
 
+    @Override
+    public boolean removeSpecificRecords(Class<? extends BaseEntity> entityClass, List<Integer> ids) {
+        if (ids == null || ids.isEmpty()) {
+            throw new IllegalArgumentException("IDs list cannot be null or empty");
```

Comment:
Please use constant and reuse in tests. I think you can put this constant and constant from `add` method in common constants class.